### DISCUSSION TITLE
Custom Routing Should Work For Outgoing Changes From An Organization

### DISCUSSION THREAD

**Discussion by foohyfooh** [community/community#30544] [Other Feature Feedback, Questions, & Ideas]
While custom routing allows you to receive an email on depending on the organization that owns the repository, this only applies if the organization owns the repository that is being modified. So if I make a change in org/repo1 I will get the notifications correctly; but if I make a change, such as a pull request, from org/repo2-fork to update something in other-user/repo2, I get the notification in my default user email rather than the organization associated email. This should be updated so that the custom routing also applies when you make a change from an organization repository.

Summary Table If Text Is Confusing
| Action                                      | Current Behaviour              | Is This Expected | Desired Behaviour                                     |
|---------------------------------------------|--------------------------------|------------------|-------------------------------------------------------|
| Any Source --Change--> org/repo1          | Notification on custom routing | Yes              | N/A                                                   |
| org/repo2-fork --Change--> other-user/repo2 | Notification on default email  | No               | Notification on custom routing                        |
| org/repo3-fork --Change--> user-me/repo3    | Notification on default email  | No               | Notification on both custom routing and default email |

**Comment by svenssonaxel** [comment]
See #47242.
